### PR TITLE
Update test project description

### LINKED ISSUE #499: Is a unit test project a console app or a class lib?
I was following the http://dotnet.github.io/docs/core-concepts/testing/unit-testing-with-dotnet-test.html doc and I got a bit confused in the "Creating the test project" section. I couldn't tell if I was supposed to create a console app project or a class lib project (or maybe some sort of hybrid?).

The text says:

>Next, cd into the 'test' directory, and create the PrimeServices.Tests directory. CD into the PrimeServices.Tests directory and create a new project using dotnet new. Once again, dotnet new creates a console application, and you'll need to modify project.json to create a class library project by removing the buildOptions node.

So, I must be creating a class lib, right? But then it says:

> Finally, you need to set the framework node to use netcoreapp1.0, and include the required imports to get xunit to work with RC2:

Isn't that the TFM for creating apps not class libs? I thought class libs generally target netstandard, but when I tried that it didn't work.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #499 
update description of unit test project to match dotnet test behavior.

The unit test project is a console application project, but does not
contain the entry point for the application. That's in the testrunner.

/cc @danroth27 @blackdwarf

### CODE REVIEW

File: docs/core-concepts/testing/unit-testing-with-dotnet-test.md [hunk 1]
```diff
@@ -84,19 +84,18 @@ namespace Prime.Services
 
 Next, cd into the 'test' directory, and create the `PrimeServices.Tests` directory.
 CD into the `PrimeServices.Tests` directory and create a new project using `dotnet new`.
-Once again, `dotnet new` creates a console application, and you'll need to modify
-`project.json` to create a class library project by removing the `buildOptions` node.
+Once again, `dotnet new` creates a console application. Your unit test project is
+a console application, but the unit test assembly should not contain the application
+entry point. The xunit testrunner contains the entry point to run the console application.
+Therefore,y you need to modify
```
Comment (svick) [inline docs/core-concepts/testing/unit-testing-with-dotnet-test.md]: Stray "y".

**Comment by richlander** [comment]
Want to fix that one issue, @BillWagner ?

**Comment by BillWagner** [comment]
@richlander Done.

**Comment by mairaw** [comment]
I haven't read this topic before so I have some unrelated feedback to this particular change. 

Should we add an introduction on what the topic will cover before jumping into instructions?
Line 160: typo crates
Line 239: typo you'll see the that two 
When we're referring to xunit as a product like in line 135, I think we should use their product name that seems to be xUnit.net (https://xunit.github.io/docs/getting-started-dotnet-core.html)

There are some other things that we could look into it later like the use of CD as a verb or telling users they need to open a terminal or command first.

**Pull request merged**